pages/blog/tooling-update-1.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "AsyncAPI tooling update 1"
+date: 2022-04-13T18:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+  - Tooling Status
+cover: /img/posts/tooling-update.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+excerpt: 'A short update on what have happened since last update around tooling'
+featured: true
+---
+
+Another update on what happen in our tools that help developers develop EDAs and APIs! :fire: 
```

Comment:
```suggestion
Welcome to another update on what happens in our tools that help developers develop EDAs and APIs! :fire: 
```
Of course, the 'welcome' is up to you 😸 I just felt the original sentence beginning sounded choppy and abrupt.

But `happens` was a mispelling that needed to be fixed 👯